bot/models/extensions/games/wordle.py

```diff
@@ -0,0 +1,27 @@
+from sqlalchemy import Column, Integer, DateTime
+from bot import app
+from db.model import Model
+from datetime import datetime
+
+
+class Wordle(app.base, Model):
+	__tablename__ = 'wordle'
+
+	id = Column(Integer, primary_key=True)
+	member_id = Column(Integer, nullable=False, unique=True)
+	points = Column(Integer, nullable=False)
+	play_date = Column(DateTime, nullable=False)
+
+	@classmethod
+	def update_database(cls, member_id: int, points: int):
```

Comment:
As I said in the other PR, that's not how you use models. I think you should be able to adjust that from what we talked about. If not, you can ask.

Reply:
I mean I created this function just for convenience, idk